monitoring/alerting/rules/sas-job-launcher-rules.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: 1
```

Comment:
"Provisioned" resources cannot be modified by users via Grafana.  I think that's ok for dashboards since users can clone an existing dashboard, tweak it and save it with a new name.  But for alerts, this might not be optimal since the original alert will still be there and potentially trigger at the wrong time.  But we need/want to provide some OOTB alerts, so it's a tough call.